cpp/include/gar/util/data_type.h

```diff
@@ -51,6 +51,9 @@ enum class Type {
   /** UTF8 variable-length string */
   STRING,
 
+  /** List of some logical data type */
+  LIST,
```

Comment:
hi, @acezen . In C++, the logical data type is LIST; and in Spark, which we implemented previously, it is called ARRAY (https://github.com/alibaba/GraphAr/blob/main/spark/src/main/scala/com/alibaba/graphar/GraphInfo.scala#L65), should we unify this?

Reply:
fixed